apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { Post } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
+import React from 'react';
+import ChatBubble from '../../components/chat-bubble/chat-bubble';
+
+const formatDate = (date: Date) => {
+  return `${date.getHours()}:${String(date.getMinutes()).padStart(
+    2,
+    '0'
+  )} ${date.toLocaleDateString()}`;
+};
+
+export const chatBubbleList = ({ data }) => {
+  if (data) {
+    const posts = data.posts as Post[];
+    var chatBubbleList = posts.map((post) => (
+      <ChatBubble
+        isCurrentUser={data.currentUser.id == post.user.id}
+        chatDate={formatDate(post.createdTime)}
+        userPicture={avatarUrl(post.user)}
+      >
+        {post.message}
+      </ChatBubble>
+    ));
+
+    return <>{chatBubbleList}</>;
```

Comment:
In this case, I think that returning it as a `div` would be better so that we could apply styling to the container of the chat bubbles. Also, give it a className that is fitting.